Bio/Entrez/__init__.py

```diff
@@ -520,7 +520,7 @@ def _open(cgi, params=None, post=None, ecitmatch=False):
     # Equivalently, at least a third of second between queries
     params = _construct_params(params)
     options = _encode_options(ecitmatch, params)
-    delay = 0.1 if api_key else 0.333333334
+    delay = 0.1 if api_key else 0.37
```

Comment:
Can you add a comment like this here:

```python
# Using just 0.333333334 seconds sometimes hit the NCBI rate limit,
# the slightly longer pause of 0.37 seconds has been more reliable.
```

Otherwise the value of 0.37 is confusing and given the code comments earlier looks like a typo.

Reply:
Fixed in 9d39b276c4c900ce4c3e5322ebea83777240b5e3